internal/windows/shcoredll_windows.go

```diff
@@ -14,6 +14,11 @@ var (
 
 // GetDpiForMonitor https://docs.microsoft.com/en-us/windows/desktop/api/libloaderapi/nf-libloaderapi-getmodulehandlew
 func GetDpiForMonitor(monitor HMONITOR, dpiType int32, dpiX, dpiY *uint32) error {
+	err := getDpiForMonitor.Find()
```

Comment:
Is this still necessary? i.e. does the call below blow up? Or does it just return an error? If the later, please remove this check.

Reply:
*LazyProc.Call does not return an error, it just panics. Find() is necessary to ensure that it actually exists.